### ISSUE TITLE
Debian buster dependencies

### ISSUE THREAD

**Issue by skug67** [jellyfin/jellyfin#197]
Minor point -- libcurl3 was retired by Debian in the move from Stretch to Buster.  Probably want to make the package in the repository accept either libcurl3 or libcurl4 as a dependency.

**Comment by TheManchineel** [comment]
Same issue on Ubuntu 18.04 LTS:

```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 jellyfin : Depends: libcurl3 (>= 7.16.2) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

I already deleted Emby so I guess I'll have to wait, because I'm not reinstalling it.

**Comment by skug67** [comment]
download the deb: apt-get download jellyfin

Unpack deb: ar x jellyfin-*.deb (will create i.e. three files: debian-binary control.tar.gz data.tar.gz)

Unpack control archive: tar xzf control.tar.gz (will create: postinst postrm preinst prerm md5sums control)

Fix dependencies in control (use a text editor)--

Repack control.tar.gz: tar --ignore-failed-read -cvzf control.tar.gz {post,pre}{inst,rm} md5sums control

Repack deb: ar rcs jellyfin-new.deb debian-binary control.tar.gz data.tar.gz (order important! )


[Based on 
https://serverfault.com/questions/250224/how-do-i-get-apt-get-to-ignore-some-dependencies
2nd answer
]

**Comment by skug67** [comment]
Didn't mean to close (whoops)

**Comment by TheManchineel** [comment]
@skug67 Hey, thanks, that worked. It was actually quite easy once you figure it out; Emby seems to be working perfectly.

**Comment by joshuaboniface** [comment]
Issue is fixed by #193, new build incoming.

**Comment by TheManchineel** [comment]
> 
> 
> Issue is fixed by #193, new build incoming.

Wow, thanks, that was fast!
